src/js/structs/Framework.js

```diff
@@ -42,7 +43,11 @@ module.exports = class Framework extends Service {
     let url = this.get('webui_url');
 
     if (url != null && url !== '') {
```

Comment:
Correction: `this.getName()`, as we need it to take the name from the `labels.DCOS_PACKAGE_FRAMEWORK_NAME`, not just the last piece of the `appId`